src/main/java/io/github/thebusybiscuit/sensibletoolbox/core/enderstorage/STBEnderStorageHolder.java

```diff
@@ -35,7 +35,9 @@ protected STBEnderStorageHolder(EnderStorageManager manager, int frequency) {
     void loadInventory() throws IOException {
         File saveFile = getSaveFile();
         if (saveFile.exists()) {
-            String encoded = new Scanner(saveFile).useDelimiter("\\A").next();
+            Scanner scanner = new Scanner(saveFile);
+            String encoded = scanner.useDelimiter("\\A").next();
+            scanner.close();
```

Comment:
Use a try/with resource here

Reply:
here i used a close() because the try would not catch anything

Reply:
I said try with resource, not try/catch, it's specifically for closing things like that.
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html